examples/work-schedule-management-with-calendar/work-schedule-management-with-calendar.md

```diff
@@ -1,10 +1,10 @@
-# Work Schedule Management with Google Calendar API
+# Work schedule management with Google Calendar API
 
 Sarah relies on the Google Calendar API to efficiently manage her work schedule. Her application interacts with the API for various tasks related to scheduling and organizing work-related events and meetings.
 
 ## Step 1: Import connector
 
-Import the `ballerinax/googleapis.gcalendar` module into the Ballerina project.
+Import the `ballerinax/googleapis.gcalendar` package.

```

Comment:
module

Reply:
Fixed in a new pr https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/149